.github/workflows/macos.yml

```diff
@@ -55,12 +55,31 @@ jobs:
       - name: Add the bin directory to PATH
         run: echo "$HOME/install/bin" >> $GITHUB_PATH
       - name: Check installed version
-        if: always()
+        if: ${{ !cancelled() }}
         shell: bash -l {0}
         run: source ./.github/workflows/print_versions.sh
-      - name: Run tests
+
+      - name: Run pytest with multiple workers in parallel
+        shell: bash -el {0}
+        run: |
+          export PYTHONPATH=`grass --config python_path`:$PYTHONPATH
```

Comment:
```suggestion
          export PYTHONPATH=$(grass --config python_path):$PYTHONPATH
```

Reply:
Should these be (in other PRs) ported to our other workflows? It is aligned with our other CI steps, so that they aren't diverging from the main pytest workflow.

Reply:
Yes, eventually. `$(...)` is to be preferred to using backticks. (I believe shellcheck also recommends this.).

Reply:
I think it's better to only change here for now.

Reply:
On macOS, the default shell isn’t bash. On GitHub CI, we set the shell for the step to a bash (that’s why it would work). But if shell wasn’t set, is this syntax also supported?

Reply:
sh, bash, zsh are so related to each other that in this respect this is nothing that makes a difference. Backticks works for all, but $() is recommended (see https://www.shellcheck.net/wiki/SC2006).